android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/pagerview/PagerViewViewManager.kt

```diff
@@ -1,281 +0,0 @@
-package versioned.host.exp.exponent.modules.api.components.pagerview
```

Comment:
Looks like there are many changes in this and the new `PagerViewViewManagerImpl`, so I just want to be sure — have you tested that in Expo Go?

Reply:
I wasn't able to build Expo Go yesterday due to an unrelated issue so I ended up not noticing this, but indeed this isn't working on Expo Go, it's erroring out about an `Unresolved reference: PagerViewViewManager`

Reply:
I had to migrate `react-native-pager-view` to `expoGoConfig` but everything seems to be working fine now